zebrad/src/components/tracing/component.rs

```diff
@@ -82,6 +105,7 @@ impl Tracing {
             // Using `FmtSubscriber` as the base subscriber, all the logs get filtered.
             let logger = fmt::Layer::new()
                 .with_ansi(use_color)
+                .with_writer(non_blocking)
```

Comment:
Do we need to add the writer to the `ErrorLayer` as well, or does it use the same writer as the `fmt::Layer`?

Reply:
Here too, every layer uses the writer from fmt::Layer